docs/Installing_IML_on_HPC_Storage_Sandbox.md

```diff
@@ -0,0 +1,143 @@
+[**Table of Contents**](index.md)
+
+# Installing IML on HPC Storage Sandbox
+
+![clustre](md_Graphics/installing_sm.jpg)
+
+## Prerequisites:
+Please refer to https://github.com/intel-hpdd/vagrantfiles on how to create a virtual HPC storage cluster with vagrant before attempting to install IML.
+
+## Notes:
+- You will be logging into your vagrant box as the vagrant user, which has the ability to run with root privilege. To elevate privileges to the root account, use the sudo command. The vagrant user does not require a password to run sudo. Should there ever be a need to login as root directly, the root password is also "vagrant". 
+
+    To acquire a root shell, use sudo in either one of the following invocations:
+    ```
+    sudo -s
+    ```
+    or
+    ```
+    sudo su -
+    ```
+
+## Installing IML:
+1. Verify the following vagrant plugins are installed:
+    ```
+    vagrant plugin install vagrant-shell-commander
```

Comment:
I think you want at least these:

vagrant-share (1.1.9, system)
  - Version Constraint: > 0
vagrant-shell-commander (0.3.2)
  - Version Constraint: > 0
vagrant-vbguest (0.14.2)
  - Version Constraint: > 0

Reply:
added